goosebit/__main__.py

```diff
@@ -0,0 +1,12 @@
+import logging.config
+
+import uvicorn
+
+from goosebit import app
+from goosebit.settings import config
+
+logging.config.dictConfig(config.logging)
+
+
+if __name__ == "__main__":
+    uvicorn.run(app, host="0.0.0.0", port=config.port)
```

Comment:
So `main.py` can be removed, no?

Reply:
It can be, we may want to keep it for testing, but it is essentially a duplicate now.

Reply:
I'm going to leave it in for now, but can be removed later if we decide to.